assets/app/view/game/round/auction.rb

```diff
@@ -108,7 +108,10 @@ def render_companies
             },
           }
 
-          @step.available.select(&:company?).map do |company|
+          companies = @step.available.select(&:company?)
+          companies = companies.partition { |c| c == @step.auctioning }.flatten(1)
```

Comment:
what is this code doing

Reply:
During the bidding of privates in 1817 all the 11 privates are shown, but we are only bidding on one of them. This is putting the one we're currently bidding on the beginning.

Reply:
too much movement, i don't think wee need to do that revert this change

Reply:
I think that we really need to do something. Everyone is always complaining that it's hard to find where is the private that it's up for auction. There is barely any visual indication. This just puts one corp to the top when it's auctioned. In all other comparable auctions you don't even see the other entities.